packages/synthetics-sdk-broken-links/src/link_utils.ts

```diff
@@ -73,13 +75,50 @@ export function setDefaultOptions(
     per_link_options: {},
   };
 
+  const output_options: BrokenLinksResultV1_BrokenLinkCheckerOptions =
+    {} as BrokenLinksResultV1_BrokenLinkCheckerOptions;
   const objKeys = Object.keys(default_options) as Array<
     keyof BrokenLinksResultV1_BrokenLinkCheckerOptions
   >;
   for (const key of objKeys) {
-    if (!(key in options) && key !== 'origin_url') {
-      /* eslint-disable */
-      (options as any)[key] = default_options[key];
-    }
+    // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+    (output_options as any)[key] =
+      !(key in options) || key === 'per_link_options'
```

Comment:
maybe leave a comment explaining that per_link_options gets special "pass through" treatment because it gets handled in code that's farther down.